numba_dpex/core/parfors/parfor_lowerer.py

```diff
@@ -176,45 +108,13 @@ def _submit_parfor_kernel(
 
         local_range = []
```

Comment:
A comment is warranted here: "For now the `local_range` is always an empty list as numba_dpex always submits kernels generated for parfor nodes as `range` kernels. The provision is kept here if in future there is newer functionality to submit these kernels as `ndrange`."